torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -415,13 +415,13 @@ def apply_compile(model, job_config: JobConfig):
             "fused_rmsnorm not yet compatible with torch.compile. Please use layernorm or rmsnorm."
         )
 
+    # TODO(anijain): the following flag is on to accelarate compilation
+    #                remove it after it's enabled in pytorch by default
+    torch._dynamo.config.inline_inbuilt_nn_modules = True
```

Comment:
I think we should not enable this by default before dynamo fix 2D compile recompilation problem as @bdhirsh pointed out